processor/routingprocessor/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/processor/routi
 go 1.18
 
 require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage v0.59.0
```

Comment:
This needs to be bumped now that 0.60.0 is out

Reply:
done